ui/public/i18n/language.json

```diff
@@ -243,7 +246,10 @@
     "dnsHostRecord_do_not_be_removed": "The host name alias in System > Dashboard > Hostname / Alias is preserved",
     "automatic_HostRecord_creation": "An host name alias is added to System > Dashboard > Hostname / Alias ",
     "advanced_mode": "Advanced settings",
-    "ValidFromCIDR_help": "One item per line"
+    "ValidFromCIDR_help": "One item per line",
+    "Enable_WebSockets": "Enable WebSockets",
+    "WebSocketsPath":"WebSockets path",
```

Comment:
```suggestion
    "Enable_WebSockets": "Enable web socket",
    "WebSocketsPath":"Web socket path",
```

Reply:
There's no such thing like `web socket`, the protocol is called `WebSocket`.
The current name is the correct one, see also https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API

Reply:
Ok for the single word with capitals, however I'd prefer the singular form because it is the name of a protocol.